### PR TITLE
Do not swallow AttributeError inside index() on template rendering

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

**Comment by hvelarde** [comment]
@datakurre @mauritsvanrees Travis TI is failing again and I have no idea why; any help will be much appreciated.

File: plone/tiles/esi.py [hunk 1]
```diff
@@ -76,10 +76,12 @@ def __call__(self, *args, **kwargs):
                 queryString=self.request.get('QUERY_STRING', ''),
                 esiMode=mode
             )
+        # Do not hide AttributeError inside index()
         try:
-            return self.index(*args, **kwargs)
+            self.index
```
Comment (rodfersou) [inline plone/tiles/esi.py]: could be a `safe_hasattr(self, 'index')` instead

**Comment by datakurre** [comment]
@hvelarde Unfortunately, I'm currently sick, which will delay the merge and release (if that depends on me).

**Comment by hvelarde** [comment]
@datakurre don't worry and get well.

**Comment by gforcada** [comment]
@hvelarde if you look at travis output (https://s3.amazonaws.com/archive.travis-ci.org/jobs/212205097/log.txt) you will see that is a zc.buildout version conflict, as simple (or complex) as that :-)

**Comment by gforcada** [comment]
@hvelarde I just fixed the problem if you rebase on top of master travis will be happy.

**Comment by hvelarde** [comment]
a new release would be appreciated :-)

**Comment by mauritsvanrees** [comment]
I was looking at maybe making a release.

@datakurre Plone coredev 5.0 and 5.1 both use a checked out master branch, and have version 1.8.2 in their versions.  According to the changelog, there are backwards incompatible changes on master.  Jenkins is still happy with master on both coredevs.

I am inclined to say: coredev 5.0 should use the 1.x branch, and 5.1 can use the 2.x versions from master.  Does that make sense?

**Comment by datakurre** [comment]
I should be able to do releases tomorrow. 2.x changes were introduced long after Plone 5.1 was already beta ready so, I believe, it would be correct to keep Plone 5.1 to use 1.x and only switch to master / 2.x for 5.2.

**Pull request merged**